app/src/main/java/com/github/se/wanderpals/ui/screens/dashboard/DashboardStopWidget.kt

```diff
@@ -0,0 +1,226 @@
+                              }
+
+                          Spacer(modifier = Modifier.padding(4.dp))
+
+                          Text(
+                              text =
+                                  "Total: ${sortedStops.size} stop" +
+                                      if (sortedStops.size > 1) "s" else "",
+                              modifier =
+                                  Modifier.testTag("totalStops")
+                                      .clip(RoundedCornerShape(4.dp))
+                                      .background(MaterialTheme.colorScheme.surface)
+                                      .padding(horizontal = 8.dp, vertical = 4.dp),
+                              style =
+                                  TextStyle(
+                                      color = MaterialTheme.colorScheme.primary,
+                                      fontWeight = FontWeight.Bold))
+                        }
+
+                    Spacer(modifier = Modifier.padding(4.dp))
+
+                    // Latest expenses
+                    Box(
+                        modifier =
+                            Modifier.clip(RoundedCornerShape(4.dp))
+                                .background(MaterialTheme.colorScheme.surface)
+                                .fillMaxWidth()) {
+                          if (sortedStops.isEmpty()) {
+                            Box(
+                                contentAlignment = Alignment.Center,
+                                modifier =
+                                    Modifier.padding(top = 16.dp, bottom = 40.dp).fillMaxSize()) {
+                                  Text(
+                                      text = "No stops yet.",
+                                      modifier = Modifier.testTag("noStops"),
+                                      style = TextStyle(color = MaterialTheme.colorScheme.primary),
+                                  )
+                                }
+                          } else {
+                            Column {
+                              StopItem(stop = sortedStops[0])
+                              HorizontalDivider(
+                                  color = MaterialTheme.colorScheme.surfaceVariant,
+                                  thickness = 1.dp,
+                                  modifier = Modifier.padding(horizontal = 8.dp))
+                              if (sortedStops.size > 1) {
+                                StopItem(stop = sortedStops[1])
+                              } else {
+                                Box(modifier = Modifier.fillMaxSize())
+                              }
+                            }
+                          }
+                        }
+                  }
+            }
+      }
+}
+
+/**
+ * Composable function to display a single stop item in the dashboard widget
+ * @param stop Stop object to display
+ */
+@Composable
+fun StopItem(stop: Stop) {
```

Comment:
Rename the composable as there is another composable called StopItem used in the stops list and the agenda.

Reply:
Done